packages/playground/vite.config.ts

```diff
@@ -3,7 +3,7 @@ import solidPlugin from 'vite-plugin-solid';
 import UnoCSS from 'unocss/vite';
 
 export default defineConfig((env) => ({
-  plugins: [solidPlugin(), UnoCSS()],
```

Comment:
the order matters here - this is aligned with the docs:
https://unocss.dev/integrations/vite#solid

The sr-only tags will break otherwise, exposing all the text made for screen readers